deploy/charts/rook-ceph-cluster/templates/ingress.yaml

```
@@ -1,4 +1,4 @@
-{{- if .Values.ingress.dashboard.host -}}
+{{- if .Values.ingress.dashboard.enabled -}}
```

With the new suggested approach, we could keep the `host` condition for whether to create the ingress.
```suggestion
{{- if .Values.ingress.dashboard.host -}}
```